### ISSUE TITLE
Clicking the "3 dots" when selecting path breaks filesystem browsing

### ISSUE THREAD

**Issue by kambala-decapitator** [jellyfin/jellyfin-web#4469]
**Describe The Bug**

I tried adding a Music path during initial configuration, and when clicking the "3 dots" to go to parent directory filesystem browser simply disappears. Can reproduce with Latin only characters in path as well.

Looks the same as #3998

**Steps To Reproduce**

1. Go to adding new path
2. Open some directory
3. Click the "3 dots"

**Expected Behavior**

Parent directory is opened

**Logs**

```
[2023-04-01 16:22:02.214 +03:00] [INF] [4] Main: Jellyfin version: "10.8.9"
[2023-04-01 16:22:02.252 +03:00] [INF] [4] Main: Environment Variables: ["[JELLYFIN_LOG_DIR, /Users/kambala/.local/share/jellyfin/log]"]
[2023-04-01 16:22:02.254 +03:00] [INF] [4] Main: Arguments: ["/Applications/Jellyfin.app/Contents/MacOS/jellyfin.dll", "--webdir", "/Applications/Jellyfin.app/Contents/Resources/jellyfin-web"]
[2023-04-01 16:22:02.256 +03:00] [INF] [4] Main: Operating system: "macOS"
[2023-04-01 16:22:02.256 +03:00] [INF] [4] Main: Architecture: X64
[2023-04-01 16:22:02.256 +03:00] [INF] [4] Main: 64-Bit Process: True
[2023-04-01 16:22:02.256 +03:00] [INF] [4] Main: User Interactive: True
[2023-04-01 16:22:02.257 +03:00] [INF] [4] Main: Processor count: 6
[2023-04-01 16:22:02.257 +03:00] [INF] [4] Main: Program data path: "/Users/kambala/.local/share/jellyfin"
[2023-04-01 16:22:02.257 +03:00] [INF] [4] Main: Web resources path: "/Applications/Jellyfin.app/Contents/Resources/jellyfin-web"
[2023-04-01 16:22:02.257 +03:00] [INF] [4] Main: Application directory: "/Applications/Jellyfin.app/Contents/MacOS/"
[2023-04-01 16:22:02.272 +03:00] [INF] [4] Jellyfin.Server.Migrations.MigrationRunner: Marking following migrations as applied because this is a fresh install: ["CreateNetworkConfiguration"]
[2023-04-01 16:22:02.342 +03:00] [INF] [4] Emby.Server.Implementations.AppBase.BaseConfigurationManager: Setting cache path: "/Users/kambala/.cache/jellyfin"
...
[2023-04-01 16:23:33.389 +03:00] [WRN] [39] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from "http://localhost:8096/Environment/DirectoryContents?includeDirectories=true&path=%2FVolumes%2FextHDD" to "127.0.0.1" in 0:00:01.8991278 with Status Code 200
[2023-04-01 16:23:50.800 +03:00] [ERR] [47] Jellyfin.Server.Middleware.ExceptionMiddleware: Error processing request: "Could not find a part of the path '/\"/Volumes/extHDD/music/\u0411\u0438\u0431\u043B\u0438\u043E\u0442\u0435\u043A\u0430/Artist-Album\"'". URL "GET" "/Environment/DirectoryContents".
[2023-04-01 16:24:19.202 +03:00] [ERR] [44] Jellyfin.Server.Middleware.ExceptionMiddleware: Error processing request: "Could not find a part of the path '/\"/Volumes/extHDD/music/\u0411\u0438\u0431\u043B\u0438\u043E\u0442\u0435\u043A\u0430/Artist-Album/A-Ha\"'". URL "GET" "/Environment/DirectoryContents".
[2023-04-01 16:54:05.117 +03:00] [ERR] [83] Jellyfin.Server.Middleware.ExceptionMiddleware: Error processing request: "Could not find a part of the path '/\"/Volumes/extHDD/music/\u0411\u0438\u0431\u043B\u0438\u043E\u0442\u0435\u043A\u0430\"'". URL "GET" "/Environment/DirectoryContents".
```

**Screenshots**

video:

https://user-images.githubusercontent.com/1557784/229293729-273b2a55-896d-4bed-9f0d-b35c72665374.mov

**System (please complete the following information):**
 - Platform: macOS 12.6.4
 - Browser: Safari 16.3 (17614.4.6.11.6, 17614)
 - Jellyfin Version: 10.8.9]

**Comment by kambala-decapitator** [comment]
still the case. Tested in Safari 16.6 on macOS, server is 10.8.10-1 on Debian.

**Comment by chriscavalluzzi** [comment]
Hi there! I think this bug was fixed in version 10.8.11. I was able to reproduce it in 10.8.10, but I was not able to reproduce it on 10.8.11 or `master` (as of today) with either Latin-only or non-Latin characters.

**Comment by kambala-decapitator** [comment]
I also confirm that on 10.8.11 the issue is fixed
